d3-array/d3-array-tests.ts

```diff
@@ -330,12 +330,23 @@ mergedArray = d3Array.merge(testArrays); // inferred type
 mergedArray = d3Array.merge<MixedObject>(testArrays); // explicit type
 // mergedArray = d3Array.merge<MixedObject>([[10, 40, 30], [15, 30]]); // fails, type mismatch
 
+// cross() ---------------------------------------------------------------------
+
+let crossed: Array<[string, number]>;
+
+crossed = d3Array.cross<string, number>(['x', 'y'], [1, 2]);
```

Comment:
While I agree that a separate shape test could simply use inference, it does not fully do away with the test explicitly typing the generics. For example, a definition typo like the following, would not be caught with only an inference based test:
```ts
export function cross<S, T>(a: T[], b: S[]): Array<[T, S]>;
```
Notice, the "accidental argument reordering. In short, I always prefer to have a test that explicitly types. Across the different D3 modules we have some complicated scenarios with generics. So I would like to keep this consistency, even though it might not be strictly necessary here.